webgpu.h

```diff
@@ -722,6 +721,12 @@ typedef struct WGPUBindGroupLayoutDescriptor {
     WGPUBindGroupLayoutEntry const * entries;
 } WGPUBindGroupLayoutDescriptor;
 
+typedef struct WGPUBufferCopyView {
+    WGPUChainedStruct const * nextInChain;
```

Comment:
there is a bit of a problem that both structures contain `nextInChain` fields, is that intentional?

Reply:
Somewhat, because either is a "dictionary" and can be extended. That shouldn't be a problem in general, `WGPURenderPipelineDescriptor` has multiple extension points too.